subworkflows/local/prepare_reference_sequences.nf

```diff
@@ -4,21 +4,37 @@ include { CREATE_REFERENCE_FROM_GENOMES } from '../../modules/local/create_refer
 workflow PREPARE_REFERENCE_SEQUENCES {
```

Comment:
Previously, we required users to pass in files with absolute paths because that was the only way we could get Nextflow to read them. During this story, I figured out how to accept files on the command line (when I was handling the dada2.clusters.txt file), so that we did not need an absolute path. I made the change here so that we no longer needed an absolute path for Genomes. This is an issue that was brought up before and I saw this as an opportunity to make the fix globally since it was occurring only in one other place.